### PR TITLE
Single image downloads

### PR DESCRIPTION
**Pull request by gestchild**
References #4938

## Who is this for?
People who want to download single images from the viewer

## What is it doing for them?
Adds links to 2 sizes of the current image being displayed

Screenshot 2020-03-25 at 16 15 15

### CODE REVIEW

File: common/views/components/IIIFViewer/IIIFViewer.js [hunk 1]
```diff
@@ -292,8 +295,27 @@ const IIIFViewerComponent = ({
...
+  const smallImage = urlTemplateMain && {
+    '@id': urlTemplateMain({ size: '760,' }),
+    format: 'image/jpeg',
+    label: 'This image (760 pixels)',
+  };
   const iiifPresentationDownloadOptions =
-    (manifest && getDownloadOptionsFromManifest(manifest)) || [];
+    (manifest &&
+      [
+        ...getDownloadOptionsFromManifest(manifest),
```
Comment (davidpmccormick) [inline common/views/components/IIIFViewer/IIIFViewer.js]: should this be renamed now that `largeImage` and `smallImage` are also download options from the manifest?

File: catalogue/webapp/components/Download/Download.js [hunk 2]
```diff
@@ -160,9 +160,9 @@ const Download = ({
                   .filter(option => option.format !== 'text/plain') // We're taking out raw text for now
                   .map(option => {
                     const action =
-                      option.label === 'Download full size'
+                      option.label === 'This image (Full size)'
```
Comment (jamesgorrie) [inline catalogue/webapp/components/Download/Download.js]: Just wondering where the language came from? Would download on it's own make sense?

Comment (gestchild) [reply catalogue/webapp/components/Download/Download.js]: You can see the discussion on language used here: #4938

File: common/views/components/IIIFViewer/IIIFViewer.js (hunk 1 above)
Comment (gestchild) [reply common/views/components/IIIFViewer/IIIFViewer.js]: I still think it makes sense, there are just more options now.

Comment (davidpmccormick) [reply common/views/components/IIIFViewer/IIIFViewer.js]: Oh I get you. Yeah, makes sense.

**Comment by pollecuttn** [comment]
@gestchild Can I put this in the fortnightly email to the department? Is it deployed?

And can #4938 be closed?

**Comment by gestchild** [comment]
@pollecuttn yes and yes, it's now live

**Pull request merged**